packages/client/src/stats/SfuStatsReporter.ts

```diff
@@ -60,10 +120,16 @@ export class SfuStatsReporter {
       webrtcVersion: this.webRTCVersion,
       subscriberStats,
       publisherStats,
+      audioDevices: this.inputDevices.get('mic'),
+      videoDevices: this.inputDevices.get('camera'),
+      deviceState: { oneofKind: undefined },
```

Comment:
I've tried using Compute Pressure API in Chrome, I'll push this to your branch :)